solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedFeatureStore.java

```diff
@@ -77,33 +83,28 @@ protected void onManagedDataLoadedFromStorage(NamedList<?> managedInitArgs,
       @SuppressWarnings("unchecked")
       final List<Map<String,Object>> up = (List<Map<String,Object>>) managedData;
       for (final Map<String,Object> u : up) {
-        update(u);
+        addFeature(u);
       }
     }
   }
 
-  public void update(Map<String,Object> map) {
-    try {
+  public void addFeature(Map<String,Object> map) {
```

Comment:
We can have 1, but this seems like more the Java style for doing the default param like in C++